CMR/python/demos/scripts/demo.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+"""A simple test of the API from a command line script"""
+
+import time
+
+try:
+    # Try loading the code from the python environment
+    import cmr as cmr_imp
+except ModuleNotFoundError:
+    # Try to load the local system
+    import os
+    import sys
+    sys.path.append(os.path.expanduser('.'))
+    import cmr as cmr_imp
+import cmr.search.collection as coll
+
+# Terminal colors
+def text_color(message='{}', color_code='\033[0;37m'):
+    """Set text to a color, default color is White"""
+    no_color = '\033[0m'
+    return '{}{}{}'.format(color_code, message, no_color)
+
+def red_text(msg='{}'):
+    """Set text to red"""
+    red_code = '\033[0;31m'
+    return text_color(msg, red_code)
+
+def green_text(msg='{}'):
+    """Set text to green"""
+    green_code = '\033[0;32m'
+    return text_color(msg, green_code)
+
+def blue_text(msg='{}'):
+    """Set text to blue"""
+    blue_code = '\033[0;34m'
+    return text_color(msg, blue_code)
```

Comment:
It's only these three functions now but you might want to print text in more colors later, in which case you will have a lot of functions, like purple_text, cyan_text, yellow_text. It might be better to handle this with a dictionary where the keys are the color names in English (i.e. "purple") and the values are the codes (i.e. '\033[0;34m')? There's the https://pypi.org/project/termcolor/ Python module but you mentioned you don't want this project to use too many external libraries because that might cause NASA unexpected problems, so maybe dictionaries would be a better way to go

Reply:
I don't want to support more colors, maybe I could rename the functions to say what I want to about them, I used the color to denote action, vs header, vs result

Reply:
rename the functions to input->blue, output->green, error->red

Reply:
renamed these, check it out.